common/fs/src/tail.rs

```diff
@@ -204,7 +204,7 @@ impl Tailer {
                             // if Some(..) = entries.get(key) clauses)
                             let mut entries = fs.entries.borrow_mut();
                             if entries.remove(entry_ptr).is_some() {
-                                debug!(
+                                info!(
```

Comment:
Should this be put back to debug? If we'd like to keep it at info the 
message should be tweaked to not talk about the map, as that's an implementation detail that doesn't need logged

Reply:
I see what you mean, having it an info level can be useful but the message doesn't make much sense for the user, how about: `"Removed file information, currently tracking {} files"`

Reply:
That sounds good to me!